content/news/2020-06-08-reinventing-techs-pipeline.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Reinventing tech's pipeline: Inside the effort to boost black and Latinx talent"
+date: 2020-06-08
+image: "/assets/img/news/2020-06-08-protocol"
```

Comment:
This needs `.jpg` at the end in order to render properly

Reply:
updated